microsoft-edge/devtools-guide-chromium/whats-new/2023/06/devtools-114.md

```diff
@@ -20,174 +20,162 @@ ms.date: 05/31/2023
 
-<!-- Subtitle: Reflow is when the browser re-calculates the positions and geometries of elements in the page and is user-blocking so limit reflow as much as possible for better performance.-->
+<!-- Subtitle: The Styles pane shows a wavy underline on CSS properties that trigger reflow, and the Issues tool warns about such properties. This notification helps limit re-calculating positions and geometries of elements, to avoid blocking user interaction. -->
 
 <!-- Reviewer: Vidal Guillermo Diazleal Ortega -->
 
-Reflow is the name of the web browser process for re-calculating the positions and geometries of elements in the document, for the purpose of re-rendering part or all of the document. Because reflow is a user-blocking operation in the browser, from a performance perspective, we recommend limiting reflow as much as possible to keep your web content smooth and interactable. Some CSS properties don't trigger reflow because they run on the compositor thread in the browser, like [transform](https://developer.mozilla.org/docs/Web/CSS/transform) and [opacity](https://developer.mozilla.org/docs/Web/CSS/opacity).
+DevTools now detects and warns about properties that can cause performance issues when animated.
+*  In the **Styles** pane in the **Elements** tool, a wavy underline and tooltip has been added on CSS properties that trigger reflow.
+*  In the **Issues** tool, in the **Performance** category, a new kind of issue is reported, such as "Changes to this property will trigger: 'Layout', which can impact performance when used inside @Keyframes."
 
-Now, the **Styles** pane in the **Elements** tool adds a wavy underline for CSS properties that do trigger reflow.  Hovering over the wavy underline will provide a tooltip that shows the issue
+Because reflow is a user-blocking operation in the browser, for performance, we recommend limiting reflow as much as possible, to keep your web content smooth and responsive to interaction.  Reflow is the web browser process for recalculating the positions and geometries of elements in the document, to re-render part or all of the document.  Some CSS properties don't trigger reflow, because they run on the compositor thread in the browser - for example, the [transform](https://developer.mozilla.org/docs/Web/CSS/transform) and [opacity](https://developer.mozilla.org/docs/Web/CSS/opacity) properties.
+
+In the **Styles** pane, to view the issue in a tooltip, hover over the wavy underline:
 
 ![CSS reflow issue in the Styles pane](./devtools-114-images/css-reflow-issue-styles-pane.png)
 
-You can `Shift`+`click` the wavy underline to open the issue in the **Issues** tool
+To open the issue in the **Issues** tool, **Shift**+click the wavy underline, or right-click the wavy underline and then select **View issues**.  The **Issues** tool opens in the Drawer at the bottom of DevTools:
 
 ![CSS reflow issue in the Issues tool](./devtools-114-images/css-reflow-issue-issues-tool.png)
```

Comment:
Need to replace this image.

Reply:
that's re: the 2nd png in section:
https://github.com/mikehoffms/edge-developer/blob/user/mikehoffms/whats-new-114/microsoft-edge/devtools-guide-chromium/whats-new/2023/06/devtools-114.md#the-issues-tool-and-styles-pane-warn-about-css-properties-that-trigger-reflow